### PR TITLE
feat(theme): update styles to support themeing

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #211

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/tokens.scss
Comment (jordanjones243) [inline src/tokens.scss:95]: background-lightest is a deprecated token

File: src/tokens.scss
Comment (jordanjones243) [inline src/tokens.scss:94]: ui-default-default is a deprecated token

File: src/tokens.scss
Comment (jordanjones243) [inline src/tokens.scss:109]: Should be using `ui-secondary-hover-inverse`

File: src/tokens.scss
Comment (jordanjones243) [inline src/tokens.scss:104]: Deprecated token

File: src/tokens.scss
Comment (jordanjones243) [inline src/tokens.scss:117]: Deprecated token, need to use new token

**Review by jordanjones243 — CHANGES_REQUESTED** [review]
Obviously there needs to be review from design, but there were a number of places (and I did not look through every single token) where there are available tokens that use the correct name and value, but are not included. That needs to be fixed before we push this to design review and release a beta.

File: demo/index.html
Comment (jason-capsule42) [reply demo/index.html:30]: I implemented this into auro-docsite for my local beta. This will get us past the theme demo for now. Once you approve this PR I will make a final edit to remove this custom style locally.

**Pull request merged**
